command/v6/services_command.go

```diff
@@ -105,3 +109,21 @@ func (cmd ServicesCommand) Execute(args []string) error {
 
 	return nil
 }
+
+func sortServiceInstances(instanceSummaries []v2action.ServiceInstanceSummary) {
+	sort.SliceStable(instanceSummaries, func(i, j int) bool {
+		return sorting.LessIgnoreCase(instanceSummaries[i].Name, instanceSummaries[j].Name)
+	})
+
+	for _, instance := range instanceSummaries {
+		sortBoundApps(instance)
+	}
+}
+
+func sortBoundApps(serviceInstance v2action.ServiceInstanceSummary) {
+	sort.Slice(
```

Comment:
Any particular reason why we use `sort.Slice` here but `sort.SliceStable` above?

Reply:
I don't think so. We've changed both to unstable, because it doesn't really matter and having `SliceStable` implies we need stability.